source/docs/articles/sites/database.md

```diff
@@ -1,7 +1,18 @@
 ---
 title: MySQL Database
 description: Learn about the Database that runs in your site
-category:
-- developing
-draft: true
+keywords: mysql, sql, database, db, databases
 ---
+The Pantheon platform provides each site a dedicated MySQL container, which can be maintained remotely or locally depending on your workflow. For a comprehensive list of MySQL settings [access your database](/docs/articles/local/accessing-mysql-databases/#database-connection-information) and use the [SHOW VARIABLES](http://dev.mysql.com/doc/refman/5.0/en/show-variables.html) statement.
+
+## Maintain Database Content and Configurations
+The [Pantheon Workflow](/docs/articles/sites/code/using-the-pantheon-workflow/) moves content down from your Live environment to Test and then to Dev. This means that any content added via the database to your Live site can be imported into the existing database on the Test and Dev environments using the Workflow tool.
```

Comment:
Rephrase. Using the default workflow, you can overwrite your dev or test db's with live's. In the second sentence, clarify not just content comes.